---
layout: page
title: About
permalink: /about/
---




I am a current <a href="https://www.sph.emory.edu/departments/bios/index.html">Emory Biostatistics</a> PhD student in <a href="https://yanglab-emory.github.io/">YangLab</a>.

I joined YangLab in February of 2020 to work on a project applying the YangLab-developed <a href="https://github.com/yanglab-emory/TIGAR">Transcriptome-Integrated Genetic Association Resource (TIGAR)
</a> software tool to the newly released version 8 of the <a href="https://www.gtexportal.org/home/">Genotype-Tissue Expression (GTEx) Project</a> dataset.



My job was to train tissue-specific models for predicting genetically-regulated gene expression from genetic data for 49 different tissue types. Troubleshooting during that process eventually led me to re-write most of the code, improving both speed and memory usage of the tool; the TIGAR-V2 and the trained model weights paper can be found at

[*TIGAR-V2: Efficient TWAS tool with nonparametric Bayesian eQTL weights of 49 tissue types from GTEx V8.* HGG Advances, 2022. Volume 3, Issue 1.]<a href="https://doi.org/10.1016/j.xhgg.2021.100068"> </a>


